demo-snippet.html

```diff
@@ -55,64 +51,61 @@
 
 <dom-module id="demo-snippet">
   <template>
-    <style is="custom-style" include="prism-theme-default"></style>
-    <style>
+    <style include="prism-theme-default">
       :host {
         display: block;
-        @apply(--shadow-elevation-2dp);
-        @apply(--demo-snippet);
+
+        box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+                    0 1px 5px 0 rgba(0, 0, 0, 0.12),
+                    0 3px 1px -2px rgba(0, 0, 0, 0.2);
+        margin-bottom: 40px;
+        @apply --demo-snippet;
       }
 
       .demo {
-        border-bottom: 1px solid var(--google-grey-300);
```

Comment:
border is still there, i just used the color since this doesn't depend on `paper-styles` anymore